microservices/Api.Pdp/Models/ProductDetailModel.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Api.Pdp.Models
+{
+    public class ProductDetailModel
+    {
```

Comment:
didn't get to pulling all of these out of the xml yet, only a few, and a few are commented out because they are not primitives